css/main.css

```diff
@@ -7,6 +12,7 @@ body {
     -o-background-size: cover;
     background-size: cover;
```

Comment:
I'm not a huge fan of the way you indented these styles on the background-image. Personally, I would leave them all at the same level of indentation.